HTML/Excersize2/script.js

```diff
@@ -0,0 +1,22 @@
+let clock = document.getElementById("clock");
+let daytime = document.getElementById("daytime");
```

Comment:
since the values doesn't change, you should have used const.

Reply:
yes added const